src/main/java/org/dependencytrack/persistence/TagQueryManager.java

```diff
@@ -72,11 +74,22 @@ public class TagQueryManager extends QueryManager implements IQueryManager {
     /**
      * @since 4.12.0
      */
-    public record TagListRow(String name, long projectCount, long policyCount, long totalCount) {
-
+    public record TagListRow(
+            String name,
+            long projectCount,
+            long policyCount,
+            long notificationRuleCount,
+            long totalCount
+    ) {
         @SuppressWarnings("unused") // DataNucleus will use this for MSSQL.
-        public TagListRow(String name, int projectCount, int policyCount, int totalCount) {
-            this(name, (long) projectCount, (long) policyCount, (long) totalCount);
+        public TagListRow(
+                final String name,
+                final int projectCount,
+                final int policyCount,
+                final int notificationRuleCount,
+                final int totalCount
+        ) {
+            this(name, (long) projectCount, (long) policyCount, (long) notificationRuleCount, (long) totalCount);
         }
```

Comment:
You can remove constructors annotated with `// DataNucleus will use this for MSSQL.` entirely since we dropped support for that RDBMS in Hyades.